utils/rpms/daos.spec

```diff
@@ -637,6 +641,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Thu Apr 3 2025 Samirkumar Raval <samirkumar.raval@hpe.com> 2.7.101-9
```

Comment:
The change dates should be synchronized between the `utils/rpms/daos.spec` and `debian/changelog` files.

Reply:
Done